values/keycloak/keycloak.gotmpl

```diff
@@ -4,44 +4,59 @@
 {{- $isSemver := regexMatch "^[0-9.]+" $consoleLoginVersion }}
 {{- $globalPullSecret := $v.otomi | get "globalPullSecret" nil }}
 
-{{- if or $v.otomi.linodeLkeImageRepository $v.cluster.defaultStorageClass }}
-global:
-  {{- if $v.otomi.linodeLkeImageRepository }}
-  security:
-    allowInsecureImages: true
-  {{- end }}
-  storageClass: {{ $v.cluster.defaultStorageClass }}
+image:
+  repository: {{ if $v.otomi.linodeLkeImageRepository }}{{ $v.otomi.linodeLkeImageRepository }}/docker/keycloak/keycloak{{ else }}quay.io/keycloak/keycloak{{ end }}
+  tag: "26.3.3"
```

Comment:
Upstream charts are often upgraded just because the app version changes, and if we set this here, we do not benefit from that.
```suggestion
```

Also this version is a newer patch than we are using currently, so please update `apps.yaml` accordingly.